sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies.py

```diff
@@ -542,6 +542,8 @@ def send(self, request):
                         continue
                 break
             except AzureError as err:
```

Comment:
Can we find a way to not catch `AzureSigningError`?

Look into conditional catches/excepts in Python.